pkg/sqlite/deprecate.go

```diff
@@ -35,10 +38,35 @@ func (d *BundleDeprecator) Deprecate() error {
 	log.Info("deprecating bundles")
 
 	var errs []error
-
+	type nameVersion struct {
+		name    string
+		version string
+	}
+	bundleInfo := make(map[string]nameVersion)
+	// Check if all bundlepaths are valid
 	for _, bundlePath := range d.bundles {
-		if err := d.store.DeprecateBundle(bundlePath); err != nil {
-			if !errors.Is(err, registry.ErrBundleImageNotInDatabase) && !errors.Is(err, registry.ErrRemovingDefaultChannelDuringDeprecation) {
+		name, version, err := d.querier.GetBundleNameAndVersionForImage(context.TODO(), bundlePath)
```

Comment:
Is `bundlePath` supposed to be a globally unique image tag, or can it be a directory path?

Reply:
It's a globally unique tag.